src/channel.cr

```diff
@@ -63,8 +63,13 @@ abstract class Channel(T)
   #
   # ```
   # channel = Channel(Int32).new
-  # channel.send(1)
+  # spawn do
+  #   channel.send(1)
+  #   channel.send(2)
+  # end
   # channel.receive # => 1
+  # channel.close
+  # channel.receive # raises Channel::ClosedError
```

Comment:
I do not completely understand the original code's intent,
but `channel.send(1)` without `spawn` locks the operation.

Therefore, I modified the example code based on the following two points as the docs says.
1. works well when the channel is active
2. raises when the channel is not active

Reply:
I would prefer to leave this change out of the PR. I agree that the sample is not the best, but let’s wait for some upcoming changes in the channel. The rest of the PR looks good as usual 🎩

Reply:
Yep. Just revert? or put some comments like this?

```
channel = Channel(Int32).new
channel.send(1) # TODO: This blocks
channel.receive # => 1
```

Reply:
When there is a sample code, it should work. Either we fix this in some way or remove it.
Maybe just use a buffered channel?

Reply:
Just revert the changes in this file for this PR please.

Reply:
Reverted. I hope Brian will write cool example or implementation soon :)